addons/account/migrations/9.0.1.1/pre-migration.py

```diff
@@ -212,6 +217,31 @@ def blacklist_field_recomputation(env):
     ]
 
 
+def merge_supplier_invoice_refs(env):
```

Comment:
This method in pre or post??

Reply:
It can fit in both, but better soon than later.